src/renderers/shaders/ShaderChunk/clipping_planes_fragment.glsl

```diff
@@ -0,0 +1,11 @@
+#if NUM_USER_CLIPPING_PLANES > 0
+
+	for ( int i = 0; i < NUM_USER_CLIPPING_PLANES; ++ i ) {
+
+		vec4 plane = userClippingPlanes[ i ];
+		if ( dot( vViewPosition, plane.xyz ) > plane.w ) discard;
```

Comment:
> Feel free to add "signedPlaneDistance" to somewhere here:

I prefer to feel even freer to leave it as is, because there is no more concise way to precisely specify the semantics than with a simple dot product. Citing the manual page of `glClipPlane`:

> If the dot product of the eye coordinates of a vertex with the stored plane equation components is positive or zero, the vertex is in with respect to that clipping plane. Otherwise, it is out.

So is this side the "negative half space", or how is it called? This way, I don't have to worry :-)